### PR TITLE
Table: Update selected row color

### PR DESCRIPTION
**Pull request by gtk-grafana**
**What is this feature?**
The current color 

Proposed:
image
image
image

Current:
image
image
image

**Why do we need this feature?**
Really hard to see which log line is currently selected.

**Who is this feature for?**
Users of table/ explore logs

**Special notes for your reviewer:**
Should we update the colors of the `theme.components.table.rowHoverBackground` to have a bit more contrast instead of using an existing color in our theme? Or is there something else we should use? Would love to hear your thoughts @nmarrs  @staton-hyse11 

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: packages/grafana-ui/src/components/Table/RowsList.tsx
Comment (drew08t) [inline packages/grafana-ui/src/components/Table/RowsList.tsx:274]: I think we should consider updating the `theme.components.table.rowHoverBackground` color instead of pointing to `theme.colors.background.canvas`.

Comment (gtk-grafana) [reply packages/grafana-ui/src/components/Table/RowsList.tsx:274]: Agreed.
Right now it's defined as `colors.emphasize(colors.background.primary, 0.03)`, should we just bump up the emphasize a bit? I'll post some screenshots in an hour or so after my next call

Comment (drew08t) [reply packages/grafana-ui/src/components/Table/RowsList.tsx:274]: Sounds good!

Comment (gtk-grafana) [reply packages/grafana-ui/src/components/Table/RowsList.tsx:274]: Oh ok, so the theme.components.table.rowHoverBackground is also used on hover of the rows, not just when a row is "selected", so this will change a bit more then just the feature I was targeting, but I think that's ok.


Here's `colors.emphasize(colors.background.primary, 0.1)` with the "selected row", hovering looks a bit different, I'll get to that in a bit:
Light:
image
Dark:
image

Here's 0.05
image
image

Here's 0.075
image
image


Hover seems to only change the cells in the row that you aren't hovering on: (with 0.075 value)
image
image
image
Don't think this is something we need to address in this PR, but I wanted to call it out

What do you think?

Comment (drew08t) [reply packages/grafana-ui/src/components/Table/RowsList.tsx:274]: Yeah, I noticed that too. Not sure it needs to be addressed here.

As for the emphasize value, I'll defer to @staton-hyse11's judgement!

Comment (nmarrs) [reply packages/grafana-ui/src/components/Table/RowsList.tsx]: @staton-hyse11 is back in so should be able to take a look at this / unblock this soon :)

Comment (staton-hyse11) [reply packages/grafana-ui/src/components/Table/RowsList.tsx]: Would using **action-selected** and **action-hover** work here?

Comment (gtk-grafana) [reply packages/grafana-ui/src/components/Table/RowsList.tsx:274]: Works for me, quite a bit more noticeable then what we've got now.

image
image

Not much differentiation between the two, but I think that's ok

**Review by drew08t — APPROVED** [review]

**Review by staton-hyse11 — APPROVED** [review]
LGTM!

**Pull request merged**
